.github/workflows/hardhat.yml

```diff
@@ -8,6 +8,7 @@ on:
 
 jobs:
   build:
+    name: hardhat
```

Comment:
Unrelated change, but this is the reason why the `hardhat` CI check wasn't showing up in the list of required status checks for PRs.